aten/src/ATen/native/cpu/Reduce.h

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+#include <ATen/native/cpu/Loops.h>
+#include <ATen/Parallel.h>
+
+#include <sstream>
+
+namespace at { namespace native { namespace {
```

Comment:
did you want an unnamed namespace in a header file?  (https://wiki.sei.cmu.edu/confluence/display/cplusplus/DCL59-CPP.+Do+not+define+an+unnamed+namespace+in+a+header+file).